.gitignore

```diff
@@ -156,3 +156,7 @@ bower_components/
 main/static/assets/img2
 
 *storybook.log
+
+# Data
+data/
+!data/levels.json
```

Comment:
I'm wondering why a copy of the `skills-competencies-framework.json` is excluded from version control? I may have missed previous conversations about that. I can see pros and cons, but for the moment, I think it might be useful to bundle a copy of the json file with each release of the WebApp?

Reply:
We could do that. It would definitely simplify things, although it would mean that we need to release a new version of the webapp every time there is a change to the framework. I think having a way to update the database without involving git is better. In general, not using git to manage versions of data is preferable.

It's definitely something we can do _for now_ and then address #460 later. Thoughts?

Reply:
Agree. Not a long term solution, but ok to get us through the RSECon workshop. 

Although the `populate_db.py` script is great, I had some questions/concerns about how it would operate if skills are moved within the framework or restructured etc. It might be that we can use the `populate_db.py` script for empty databases, but when tables contain any data, then use a more sophisticated approach of migrations to manage changes? 

Either way, I would be happy for you to include a temporary copy of the framework json.